src/test/regress/sql/multi_partitioning.sql

```diff
@@ -1303,7 +1303,23 @@ SELECT worker_partitioned_relation_total_size(oid) FROM pg_class WHERE relname L
  \c - - - :master_port
 DROP TABLE "events.Energy Added";
 
+-- test we skip the foreign key validation query on coordinator
+-- that happens when attaching a non-distributed partition to a distributed-partitioned table
+-- with a foreign key to another distributed table
+SET search_path = partitioning_schema;
+SET citus.shard_replication_factor = 1;
+CREATE TABLE another_distributed_table (x int primary key, y int);
+SELECT create_distributed_table('another_distributed_table','x');
+CREATE TABLE distributed_parent_table (
```

Comment:
I think a test when a default partition exists might be interesting as well. PG behaves slightly differently when default partition exists.